runtime/src/accounts_db.rs

```diff
@@ -831,6 +838,11 @@ pub struct AccountsDb {
     load_limit: AtomicU64,
 
     is_bank_drop_callback_enabled: AtomicBool,
+
+    // Set of slots currently being flushed by `flush_slot_cache()` or removed
+    // by `remove_unrooted_slot()`. Used to ensure `remove_unrooted_slots(slots)`
+    // can safely clear the set of unrooted slots `slots`.
```

Comment:
I love the comments. Could they be `///` instead?

Reply:
done!